api/src/services/fetchText.ts

```diff
@@ -1,34 +1,22 @@
-import { Request, Response } from 'express'
-import { fetchEntryText } from './fetchEntry'
-import { Knex } from 'knex'
+import graphRepository, { Entry } from './GraphRepository'
 
-export default async (req:Request, res:Response, pg:Knex, timeLogID:string) => {
-  let entry = await fetchEntryText(req.params.name, pg)
-    .catch(err => {throw err})
-  let newEntry = false
+export default async (name: string): Promise<Entry> => {
+    let entry = await graphRepository.findByName(name)
 
-  if (entry.name === '') {
-    const addEntry = async () => {
-      await pg('graphs').insert({
-        'text': '',
-        'img': '',
-        'name': req.params.name,
-        'outputUpdated': false
-      }).then((item:object) => {
-        console.timeLog(timeLogID)
-        console.info(`${timeLogID}: NEW ITEM: \"${req.params.name}\"`)
```

Comment:
Kurde, przyznaj się.
Robisz te console info wszędzie, tylko dlatego, bo nie masz testów, które sprawdzają, czy te klasy na pewno działają?

Reply:
Nie, po prostu stwierdziłem że fajnie to będzie w logach wyglądać.

Reply:
To pokażę jak to zrobić z pomocą tego komentarza:
https://github.com/hspsh/dcr/pull/4#discussion_r794673518

Bo według mnie, to bardzo zanieczyszcza kod i utrudnia jego czytanie.